oe_corporate_site_info.module

```diff
@@ -97,6 +103,12 @@ function _oe_corporate_site_info_settings_form_submit(&$form, FormStateInterface
   $corporate_site_info = \Drupal::configFactory()->getEditable('oe_corporate_site_info.settings');
   $corporate_site_info->set('site_owner', $form_state->getValue('site_owner'));
   $content_owners = $form_state->getValue('content_owners', []);
-  $corporate_site_info->set('content_owners', $content_owners);
+  $content_owner_ids = [];
+  foreach ($content_owners as $content_owner) {
+    if (!empty($content_owner['target'])) {
+      $content_owner_ids[] = $content_owner['target'];
+    }
+  }
```

Comment:
The multivalue_form_element already filters out all the empty values, this can be simplified with
```suggestion
  $content_owner_ids = array_column($form_state->getValue('content_owners', []), 'target');
```
If you really think that extra checks must be there, just pass the array to an array_filter.